pyproject.toml

```
@@ -36,3 +45,24 @@ addopts = "--ignore=test/integration_tests/resources"
 testpaths = [
     "test"
 ]
+
+[[tool.mypy.overrides]]
+module = ["exasol_integration_test_docker_environment.*", "luigi.*", "docker.*",
+            "humanfriendly", "configobj", "exasol.bucketfs", "toml"]
+ignore_missing_imports = true
+
+
+[tool.pylint.'MESSAGES CONTROL']
+max-line-length = 120
+disable = """
+    W,R,C,
+    duplicate-code,
+    missing-class-docstring,
+    missing-module-docstring,
+    missing-function-docstring,
+    too-many-ancestors,
+    broad-exception-raised,
+    broad-exception-caught,
+    consider-using-f-string,
+    useless-parent-delegation
+"""
```

Same as above:
start with an empty list, or at least:
```suggestion
disable = """
    W,R,C,
"""
```
and then add what is really necessary.